test/pom.xml

```diff
@@ -175,5 +188,13 @@
 				<selenium.session.browser>chrome</selenium.session.browser>
 			</properties>
 		</profile>
+		<profile>
+			<id>headlesschrome</id>
+			<properties>
+				<selenium.session.browser>headlesschrome</selenium.session.browser>
+				<selenium.chrome.window.width>1920</selenium.chrome.window.width>
```

Comment:
It really needs window dimensions?

Reply:
the way the ep-selenium-framework utilizes it does. it's essentially the only difference. the reason is because when using window.maximize()/fullscreen() in headless, it doesn't know what the max size would be.

Reply:
Any reason you chose something higher res? Why not 600 x 400? haha